tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -244,16 +279,18 @@ TangoErrorType TangoRosNode::TangoSetupConfig() {
 
   tango_config_ = TangoService_getConfig(TANGO_CONFIG_DEFAULT);
   if (tango_config_ == nullptr) {
-    LOG(ERROR) << function_name << ", TangoService_getConfig error.";
+    Log(ERROR, std::ostringstream().flush() <<
```

Comment:
That would be best indeed, maybe I am missing something but I don't see how.
I am using flush to create a temporary `std::ostream&` that I need to make messages using the `<<` operator. 
More info here: http://stackoverflow.com/questions/8287188/stdostringstream-printing-the-address-of-the-c-string-instead-of-its-content

Another possibility would be to use `std::string` instead of `std::ostream`, using strings concatenation with the `+` operator. But that would require to convert any non-string value to string.